nvm.sh

```diff
@@ -274,37 +274,46 @@ nvm_tree_contains_path() {
 # Traverse up in directory tree to find containing folder
 nvm_find_up() {
   local path
+  local file
   path="${PWD}"
-  while [ "${path}" != "" ] && [ ! -f "${path}/${1-}" ]; do
-    path=${path%/*}
+
+  # Iterate through the multiple files
+  while [ $# -ne 0 ]; do
+    # Look for files in turn in this path
+    while [ "${path}" != "" ]; do
+      # Is the file here?
+      if [ ! -f "${path}/${1}" ]; then
+        file="${path}/${1}"
+        echo $file
+        break 2
```

Comment:
this seems like it will echo the file twice - instead, should this `return 0`?

Reply:
It used to echo when it found it and now it echos when it finds it. I can return instead of breaking, but it didn't originally return.

Reply:
hm, that would imply that it always echoed twice, which seems weird